File: yandex_backend/2. Уникальные элементы/solution_.py
import sys
from typing import List

def main() -> None:
    n: int = int(input())
    a: List = list(map(int, input().split()))
    
    a.sort()

    count: int = 0
    i = 0
    while i < n:
        j = i
        while j < n and a[i] == a[j]:
            j += 1
        if j - i == 1:
            count += 1
        i = j

    print(count)

if __name__ == "__main__":
    main()

File: yandex_backend/2. Уникальные элементы/solution.cs
using System;
using System.Collections.Generic;
using System.Linq;

public class Solution
{
    public static void Main(string[] args)
    {
        int n = int.Parse(Console.ReadLine());
        List<int> a = Console.ReadLine().Split().Select(int.Parse).ToList();
        
        a.Sort();

        int count = 0;
        int i = 0;
        while (i < n)
        {
            int j = i;
            while (j < n && a[i] == a[j])
            {
                j++;
            }
            if (j - i == 1)
            {
                count++;
            }
            i = j;
        }
        
        Console.WriteLine(count);
    }
}